### PR TITLE
Download sources from safe sources

### PR DESCRIPTION
**Pull request by AdrianArroyoCalle**
This pull request refers to: https://codein.withgoogle.com/dashboard/task-instances/4907009086324736/?sp-page=1

There are a lot of recipes changed.
Numptyphysics doesn't work and maybe gyp is also broken (but not tested)

I've not removed the Simutrans paksets nightlies, they're newer.

### CODE REVIEW

File: dev-util/app2png/app2png-1.0.0.recipe [hunk 1]
```diff
@@ -5,10 +5,12 @@ save them into PNG files. These PNG icons can then be used to recreate the \
 icon as a vector, or for use on a website."
 
 HOMEPAGE="https://github.com/HaikuArchives/app2png"
-SOURCE_URI="git+https://github.com/HaikuArchives/app2png#17c36271a80a822ac0dfec66527e61f21230a412"
+SOURCE_URI="https://github.com/HaikuArchives/app2png/archive/17c36271a80a822ac0dfec66527e61f21230a412.tar.gz"
+CHECKSUM_SHA256="5b95076384bf7d315749a12e26f064c341389afc5552825653ea4205c9bf9b59"
 REVISION="1"
 LICENSE="MIT"
 COPYRIGHT="2015 Puck Meerburg"
+SOURCE_DIR="app2png-17c36271a80a822ac0dfec66527e61f21230a412"
```
Comment (waddlesplash) [inline dev-util/app2png/app2png-1.0.0.recipe]: This goes underneath CHECKSUM_SHA256.

File: games-engines/eepp/eepp-0.9.5.recipe [hunk 2]
```diff
@@ -44,8 +45,7 @@ BUILD_PREREQUIRES="
 	cmd:make
 	cmd:hg
 	"
-# intentionally empty
-SOURCE_DIR=""
+SOURCE_DIR="SpartanJ-eepp-b44ee6ad7704"
```
Comment (waddlesplash) [inline games-engines/eepp/eepp-0.9.5.recipe]: Same here (and below.)

**Comment by AdrianArroyoCalle** [comment]
What the hell! It appears I've mixed some commits for this pull request which where already at HaikuPorts. Sorry for the inconvenience.

**Comment by pulkomandy** [comment]
Can you fix the pull request? I think you should rebase onto haikuports master.

**Comment by AdrianArroyoCalle** [comment]
All right, fixed

**Pull request merged**